src/runtimes/auth/auth.test.ts

```diff
@@ -113,6 +113,27 @@ describe('Auth', () => {
         assert(credentialsProvider.getCredentials('iam') === undefined)
     })
 
+    it('Handles Set Bearer credentials request when parameters sent by position', async () => {
```

Comment:
Can we add test for by-name request as well? incase VSCode implementation changes as well